tests/integration/test_packages.py

```diff
@@ -76,10 +79,11 @@ def test_packages(env_package, env_name, test_env, tmpdir):
     expected_response_data = env_data["response_expectations"]
     utils.assert_elements_from_response(response_data, expected_response_data)
 
-    client.download_and_extract_archive(completed_response.id, tmpdir)
-    source_path = tmpdir.join(f"download_{str(completed_response.id)}")
-    expected_files = env_data["expected_files"]
-    utils.assert_expected_files(source_path, expected_files, tmpdir)
+    if env_name != "git_submodule":
```

Comment:
It turns out that Cachito processes Git submodules better than Git does. A Git tarball does not include the contents of a submodule, leading to this test case failing every single time. I've added in this if statement to handle this for the time being. Any other ideas on how to deal with this issue?

Reply:
Two ideas come to my mind:
1- Create a tarball with the correct contents and host it in yet another Github repo, just for the sake of this test
2- Delete from the `tmpdir` the submodule folders in case it's a "git_submodule" test

Reply:
I've implemented idea #1!